wpeverest/user-registration
# UR-217 UR-30 Enhance - Profile Picture and File Upload

### All Submissions:






### Changes proposed in this Pull Request:
Previously the profile picture field and file upload field had the following issues:
1. All of our profile pictures and file uploads were uploaded to `/uploads/year/month` folder. This PR updates all the  `user_registration_profile_pic_url` user meta from URL of profile pictures to `attachment_id` for better profile picture referencing and uploads new files to  `/user_registration_uploads/profile-pictures` and  `/user_registration_uploads/file-uploads`.
2. If we remove the profile picture and file uploads of a specific user or delete that user then his uploaded files were not deleted.
3. Duplicate files or file with same name would replace the original files.

This PR fixes all of these issues.

### How to test the changes in this Pull Request:

1. Checkout to this branch and once deactivate and activate plugin to make sure `/user_registration_uploads/profile-pictures` directory is created.
2.  Check if all the previous profile pictures and file uploads are properly referenced and previewed in profile pictures and file uploads previews in my account.
3.  Upload, remove and update profile pictures & file uploads to check all tasks related to Profile Picture and File Uploads are working properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Profile picture on different folder.
